Math-Trainer/Math-Trainer/TrainViewController.swift

```diff
@@ -8,11 +8,32 @@
 import UIKit
 
 final class TrainViewController: UIViewController {
+    // MARK: - IBOutlets
+    @IBOutlet var buttonBack: UIButton!
+    @IBOutlet var buttonsAnswerCollection: [UIButton]!
 
     // MARK: - Properties
     var type: MathTypes = .add {
         didSet {
             print(type)
         }
     }
+
+    // MARK: - Life cycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureButtons(buttonsAnswerCollection)
+        configureButtons([buttonBack])
```

Comment:
u can call this method once 
`configureButtons(buttonsAnswerCollection + [buttonBack])`

Reply:
Yep, done, thanks!